planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -1682,22 +1682,26 @@ void AvoidanceModule::generateExtendedDrivableArea(ShiftedPath * shifted_path) c
           extended_lanelets.push_back(lanelet_at_left.get());
           lanelet_at_left = route_handler->getLeftLanelet(lanelet_at_left.get());
         }
-        auto lanelet_at_right =
-          planner_data_->route_handler->getRightLanelet(lanelet_at_left.get());
```

Comment:
@taikitanaka3 
This is the cause of the issue.
Since `lanelet_at_left` might be empty, it cause the crash when we try accessing them.
Adding `if` will check the availability of the lanelet first before accessing them.